stdlib/_typeshed/__init__.pyi

```diff
@@ -189,3 +189,23 @@ else:
     @final
     class NoneType:
         def __bool__(self) -> Literal[False]: ...
+
+# This is an internal CPython type that is like, but subtly different from, a NamedTuple
+# Subclasses of this type are found in multiple modules.
+# In typeshed, `structseq` is only ever used as a mixin in combination with a fixed-length `Tuple`
+# See discussion at #6546 & #6560
+# `structseq` classes are unsubclassable, so are all decorated with `@final`.
+class structseq(Generic[_T_co]):
+    n_fields: ClassVar[int]
+    n_unnamed_fields: ClassVar[int]
+    n_sequence_fields: ClassVar[int]
+    # First argument of the constructor is called `sequence`, but can in fact take any iterable.
+    # The optional second argument *must* be a dict, rather than any old mapping.
+    #
```

Comment:
This part of the comment repeats what the type hint says. This also applies to using `dict` as opposed to `Mapping` or a protocol.

```suggestion
```

Avoiding unnecessary comments is hard, and it was hard for me too when I was new to typeshed. See the very tl;dr comments in `tkinter/__init__.pyi` for an example of how not to write comments in typeshed :)